src/PIL/ImageFont.py

```diff
@@ -796,7 +837,12 @@ def __init__(self, font, orientation=None):
         self.orientation = orientation  # any 'transpose' argument, or None
 
     def getsize(self, text, *args, **kwargs):
-        w, h = self.font.getsize(text)
+        if not kwargs.get("__internal__"):
+            deprecate("getsize", 10, "getbbox or getlength")
+        try:
+            w, h = self.font.getsize(text, __internal__=True)
+        except TypeError:
+            w, h = self.font.getsize(text)
```

Comment:
You're allowing for the theoretical use of a font class from outside of Pillow here, yes? Not one of `ImageFont`, `FreeTypeFont` or `TransposedFont`, but something else, written before this PR, and so throws an error because it doesn't know about `__internal__`.

Reply:
Yes, that is the intention for this.